lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/IMRUDriver.cs

```diff
@@ -161,6 +164,11 @@ internal sealed class IMRUDriver<TMapInput, TMapOutput, TResult, TPartitionType>
             _serviceAndContextConfigurationProvider =
                 new ServiceAndContextConfigurationProvider<TMapInput, TMapOutput, TPartitionType>(dataSet, configurationManager);
 
+            if (lifecycleManager != null)
```

Comment:
As you have default implementations for all related constructor, it should be not null.